x/deployment/types/types.go

```diff
@@ -124,20 +128,31 @@ type Group struct {
 }
 
 // ID method returns GroupID details of specific group
-func (d Group) ID() GroupID {
-	return d.GroupID
+func (g Group) ID() GroupID {
+	return g.GroupID
 }
 
-// ValidateOrderable method checks whether group opened or not
-func (d Group) ValidateOrderable() error {
-	switch d.State {
+// ValidateOrderable method checks whether group status is Open or not
+func (g Group) ValidateOrderable() error {
+	switch g.State {
 	case GroupOpen:
 		return nil
 	default:
 		return ErrGroupNotOpen
 	}
 }
 
+// ValidateClosable provides error response if group is already closed,
+// and thus should not be closed again, else nil.
+func (g Group) ValidateClosable() error {
+	switch g.State {
+	case GroupClosed:
```

Comment:
Can you make a design ticket to decide if `GroupInsufficientFunds` should be included here?

Reply:
https://github.com/ovrclk/akash/issues/684